index.html

```diff
@@ -111,7 +122,7 @@ <h2 class="tonic">Multi-Post<br>Stories</h2>
 
     <!-- card 3 -->
     <section class="work-container3">
-        <div class="portfolio-img">
+        <div class="portfolio-img desktop-img-2">
```

Comment:
[OPTIONAL] I think it will be great to confine the text to look like the Figma design instead of allowing it to stretch till the end of the card.

![optional](https://user-images.githubusercontent.com/84930355/220905730-b487cbea-f68d-4d76-b4dc-4631d6b8bf08.PNG)